### DISCUSSION TITLE
Embedding into existing page

### DISCUSSION THREAD

**Discussion by CraigHollaman** [voidlabs/mosaico#675] [Q&A] [answered]
Hi, is it possible to embedd the editor int an existing page. Does the Editor bind to a particular div element. Want to get Mosaico working inside our solution with existing navigation, side bars etc.
thanks in advance.

**Comment by bago** [comment, marked as answer]
The easy solution is to use an iframe and launch mosaico in that frame.

**Reply by CraigHollaman** [reply]
Okay. Thanks for your help.
